### PR TITLE
New Pizza Cutters

### PR DESCRIPTION
**Pull request by Mensla**
# About the pull request

Re-trying this PR as last time I got caught up with IRL stuff and let it rot.

Adds two new pizza cutter sprites, one with a plastic handle and one with a wooden handle. Adds item sprites and hand-holding sprites.

No more comically large pizza cutter as they're now appropriately sized sprites in your hand.

# Explain why it's good for the game

Allows for people to use a normal and non-memey pizza cutter. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image
image

</details>

# Changelog

:cl:
add: Added new pizza cutter sprites.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Fix the errors from the pipeline/CI

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Mensla — COMMENTED** [review]
Should be good to go, just needs reviewing now

**Review by Nanu308 — COMMENTED** [review]
mapping lgtm

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 1]
```diff
@@ -135,6 +135,44 @@
...
+/*
+ * Plastic Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter
+	name = "pizza cutter"
+	icon_state = "plasticpizzacutter"
+	desc = "A circular blade used for cutting pizzas. This one has a cheap plastic handle."
+	flags_atom = FPRINT|CONDUCT
+	sharp = IS_SHARP_ITEM_ACCURATE
+	edge = 1
```
Comment (Drulikar) [inline code/game/objects/items/tools/kitchen_tools.dm:147]:
```suggestion
	edge = TRUE
```

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 2]
```diff
@@ -135,6 +135,44 @@
...
+/*
+ * Holy Relic Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter/holyrelic
+	name = "\improper PIZZA TIME"
+	icon_state = "holyrelicpizzacutter"
+	desc = "Before you is a holy relic of a bygone era when the great Pizza Lords reigned supreme. You know either that or it's just a big damn pizza cutter."
+	sharp = IS_SHARP_ITEM_ACCURATE
+	force = MELEE_FORCE_VERY_STRONG
+	edge = 1
```
Comment (Drulikar) [inline code/game/objects/items/tools/kitchen_tools.dm:174]:
```suggestion
```
Can just be removed to be inherited from parent.

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 3]
```diff
@@ -135,6 +135,44 @@
...
+	desc = "A circular blade used for cutting pizzas. This one has an authentic wooden handle."
+
+/*
+ * Holy Relic Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter/holyrelic
+	name = "\improper PIZZA TIME"
+	icon_state = "holyrelicpizzacutter"
+	desc = "Before you is a holy relic of a bygone era when the great Pizza Lords reigned supreme. You know either that or it's just a big damn pizza cutter."
+	sharp = IS_SHARP_ITEM_ACCURATE
```
Comment (Drulikar) [inline code/game/objects/items/tools/kitchen_tools.dm:172]:
```suggestion
```
Can just be removed to be inherited from parent.

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Drulikar — APPROVED** [review]

**Review by nauticall — APPROVED** [review]

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**